nano/lib/config.cpp

```diff
@@ -306,6 +300,32 @@ std::string get_tls_toml_config_path (boost::filesystem::path const & data_path)
 }
 } // namespace nano
 
+std::optional<std::string> nano::get_env (const char * variable_name)
+{
+	auto value = std::getenv (variable_name);
+	if (value)
+	{
+		return value;
+	}
+	return {};
+}
+
+std::string nano::get_env_or_default (char const * variable_name, std::string default_value)
+{
+	auto value = nano::get_env (variable_name);
+	return value ? *value : default_value;
+}
+
+int nano::get_env_int_or_default (const char * variable_name, const int default_value)
+{
+	auto value = nano::get_env (variable_name); // 15 minutes by default
+	if (value)
+	{
+		return boost::lexical_cast<int> (*value);
```

Comment:
This will throw an exception if someone mistypes a number.
I think we should catch the exception and print an error message.
And then either abort or continue with default settings.

Reply:
Added a commit to address this.